### DISCUSSION TITLE
project Insights view time range limitation

### DISCUSSION THREAD

**Discussion by diekhans** [community/community#5196] [Other Feature Feedback, Questions, & Ideas]
It is frustrating that the Insights view only allows seeing a maximum of one-months activity.

It would very useful in understanding if a project is viable if one could sees activity over the
entire lifetime of the project.

**Comment by Mrlinkwii** [comment]
i think either increasing the number of options ( is more than month ) or letting you supply a date range with a from date and a to date from  a down arrow menu  ( ie from:<date> to : <date> ) would be a good option   thoughts @diekhans ?

**Comment by diekhans** [comment]
Increased date range would for sure help. It would be good to be able to
whole history of a project.  one day, one month, one year, entire history
seems to make sense for fixed intervals.
